rpms/README.md

```diff
@@ -5,53 +5,74 @@ needed to build a set of RPMs that are useful for working with Geotrellis.  If
 you are not intending to alter the composition or versions of the RPM
 packages, then you should be able to use the published binaries described in
 the README at the root of this project and do not need to build these
-artifacts yourself. Specifically:
+artifacts yourself.
 
-  > **Do nothing** if you are intending to deploy to EMR and have access to
-  > pre-built RPMs.  Simply point the EMR scripts to the relevant resources on
-  > S3.
+Built RPMs are publicly available at:
 
-  > **Fetch, do not build** if you have access to an already-built set of
-  > RPMs, and your purpose is to use these RPMs to build a Docker container.
```

Comment:
I would like to see a quick explanation at the top of this README as to when a user needs to to what.  Actually building these RPMs should almost never happen.  It should be 100% crystal clear, for instance, that if the intention is to build a docker image, the RPMs need to be at least fetched first.  Removing these blocks I think makes it less easy to figure out how to use the repo.  Unless they are moved to the READMEs under the `emr` and `docker` subdirectories.

Reply:
This was a terrible merge job, I'm never using the GitHub UI again. Updated to re-add the list of options that the user is presented with, and added a new "build and publish" option as well, along with links to the relevant section in this document.